README.md

```diff
@@ -1,10 +1,134 @@
-## Contribution
+# Open A**cademy**: Offering **Online Education for Data Science, Machine Learning, and Deep Learning**
 
-This open-source book has benefited from pedagogical suggestions, typo corrections, and other improvements from community contributors. Your help is valuable for making the book better for everyone.
+## Content
 
-See [this](https://github.com/open-academy/machine-learning/blob/main/CONTRIBUTING.md) for more information about how to make your contribution.
+- [Overview](https://github.com/Appointat/machine-learning/tree/main#overview)
+- [Goals](https://github.com/Appointat/machine-learning/tree/main#Goals)
+- [Features](https://github.com/Appointat/machine-learning/tree/main#Features)
+- [How to Enjoy Our Project](https://github.com/Appointat/machine-learning/tree/main#how-to-enjoy-our-project)
+- [How to Use This Project](https://github.com/Appointat/machine-learning/tree/main#How-to-Use-This-Project)
+- [How to Contribute](https://github.com/Appointat/machine-learning/tree/main#How-to-Contribute)
+- [Contact Us](https://github.com/Appointat/machine-learning/tree/main#ontact-Us)

```

Comment:
Let's have only the first letter as upper case.

Reply:
Thanks for the comment, I will modify it.